website/content/docs/ecs/install-no-terraform.mdx

```diff
@@ -0,0 +1,492 @@
+---
+layout: docs
+page_title: Manual Installation - AWS ECS
+description: >-
+  Manually Install Consul Service Mesh on AWS ECS (Elastic Container Service).
+---
+
+# Manual Installation
+
+While the [Consul ECS Terraform module](/docs/ecs/install) is the easiest way to use Consul on ECS,
+this page will describe how to directly create the ECS task definition using the [`consul-ecs` Docker image](https://gallery.ecr.aws/hashicorp/consul-ecs)
+for use without Terraform.
+
+## Pre-requisites
+
+* This page assumes you are familiar with AWS ECS. See [What is Amazon Elastic Container Service](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/Welcome.html) for more details.
+* This page does not show how to create all necessary AWS resources, such as a VPC or the ECS Cluster.
+  For complete runnable examples, see the links in the [Getting Started](/docs/ecs#getting-started) section.
+
+## Task Definition
+
+You must define a Task Definition which includes the following containers:
+
+* Your application container
+* An Envoy sidecar-proxy container
+* A Consul Agent container
+* The `consul-ecs-mesh-init` container for service mesh setup
+* Optionally, a `consul-ecs-health-sync` container to sync ECS health checks into Consul
+
+## Top-level fields
+
+In your task definition, you'll need to define these important top-level fields:
+
+```json
+{
+  "family": "my-example-client-app",
+  "networkMode": "awsvpc",
+  "volumes": [
+    {
+      "name": "consul_data",
+    },
+    {
+      "name": "consul_binary",
+    }
+  ],
+  "containerDefinitions": [...]
+}
+```
+
+| Field name             | Type   | Description                                                                                         |
+| -------------          | ------ | --------------------------------------------------------------------------------------------------- |
+| `family`               | string | The task family name. This is used as the Consul service name, by default.                          |
+| `networkMode`          | string | Must be `awsvpc`, which is the only network mode supported by Consul on ECS.                        |
+| `volumes`              | list   | These are volumes used to share information between containers, which is important for setup steps. |
+| `containerDefinitions` | list   | The list of containers to run in this task (see below).                                             |
+
+## Application container
+
+First, include your application container in the `containerDefinitions` list
+in the task definition. Ensure that the `dependsOn` field is set as shown below so that your application container starts in the correct order (see [task startup](/docs/ecs/architecture#task-startup) for more information):
```

Comment:
I've updated this with the following (pasting here for comparison, since this comment doesn't show on the newest commit).

> Ensure that the `containerName` and `condition` fields in the `dependsOn` list are specified as described in the following example. These are container dependencies, which must be used to enforce a specific [startup order](/docs/ecs/architecture#task-startup). By using the following settings, your application container will start after `consul-ecs-mesh-init` has completed task setup and after `sidecar-proxy` is ready to proxy traffic between this task and the service mesh.